dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task222208CreateVariantTable.java

```diff
@@ -0,0 +1,42 @@
+package com.dotmarketing.startup.runonce;
+
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import java.sql.SQLException;
+
+public class Task222208CreateVariantTable implements StartupTask {
+
+    private final String POSTGRES_QUERY = "CREATE TABLE IF NOT EXISTS variant ("
+            + "  id varchar(36) primary key,"
```

Comment:
No reason to be limited to 36 chars. Make it 255 at least

Reply:
done https://github.com/dotCMS/core/pull/22827/commits/3939fd3672ff479e0d14a1d8639d8ec686640682